src/app/components/calendar/calendar.css

```diff
@@ -20,6 +20,10 @@
         border-bottom-left-radius: 0;
     }
 
+    .p-calendar-icon { 
```

Comment:
Maybe I'm wrong, but it is not necessary add `cursor-pointer` and it is correct the default cursor.